### ISSUE TITLE
Open Folder: Chaotic behaviour

### ISSUE THREAD

**Issue by venturqx** [adirh3/Fluent-Search#979]
**Describe the bug**
"Open Folder" only works coherently when no Explorer window is opened. 

Depending of the number of explorer windows / number of tabs opened, sometimes :
a) Opening a random folder result in nothing
b) Opening a random folder result in another tab being remplaced by the "random folder".
c) Opening a random folder result in the current explorer window being remplaced by the "random folder" instead of opening a new window.

**To Reproduce**
Steps to reproduce the behavior (case a)):
1. Open the "AppData" folder without closing the window.
2. Open the "Downloads" folder.
3. No action is performed. The "Downloads" folder does not open.

Steps to reproduce the behavior (case c)):
1. Open the "Downloads" folder without closing the window.
2. Open the "AppData" folder.
3. The "Downloads" window is replaced by the "AppData" window.

**Expected behavior**
- Open a folder : new explorer window/tab of the folder
OR
- Open a folder : replace current explorer window/tab

**Desktop (please complete the following information):**
 - Window 11 Version: 10.0.22631 Build 22631
 - Fluent Search Version: 1.0.0.33
 - Installation source: Installer

**Comment by adirh3** [comment]
Thanks for the detailed issue!
This is most likely due to FS not recognizing when explorer is opened properly.
For now, I suggest disabling `Open folders in existing file manager window` in Settings -> Apps -> Files -> Experience.

**Comment by adirh3** [comment]
@venturqx can you please verify if this still happens in latest nightly version?

**Comment by venturqx** [comment]
_Sorry, I was unable to update to 1.0.0.36 ("Check for update" > "Update and reload" => Exit without updating). I tried to "Sync settings" in order to reinstall, but I got stuck in "Fluent Search is still loading". I had to disable the network, relaunch and disable "Sync settings" to get unstuck. I exported the settings, reinstalled and everything is fine now :)_

### 1.0.0.36
It works fine when the focused window is not Explorer :
- If an Explorer window of the researched folder is not already open : Open a new Explorer window of the researched folder 
- If an Explorer window of the researched folder is already open : Reveal the right explorer windows

However, when another Explorer window of another folder is focused :
- Instead of revealing the existing Explorer window of the researched folder if it exists : Replace the focused Explorer window's folder to the researched one

It leads to strange behaviour, e.g. :
- Search Download
- Search AppData
- Search Download
Results in 2 explorer windows of Download (with 2 different URL / view parameter, for the same folder) :
![Capture d'écran - DL](https://github.com/adirh3/Fluent-Search/assets/114839780/f53d99d0-a7fe-4777-99f3-d6a14d815ddb)

**Comment by adirh3** [comment]
@venturqx if I understand correctly, what you want is if a folder is already open in any explorer window, then bring that window/tab up even if it's not focused?

**Comment by venturqx** [comment]
@adirh3 Yes I should have said that, it's simpler. I think it's better since two windows of the same folder are never(?) needed.

**Comment by adirh3** [comment]
I will see if I can add that, might be problematic when the folder is opened but not in the focused tab
